src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -272,5 +279,704 @@ object IConvertible.ToType(Type type, IFormatProvider? provider)
         {
             return Convert.DefaultToType((IConvertible)this, type, provider);
         }
+
+#if FEATURE_GENERIC_MATH
+        //
+        // IAdditionOperators
+        //
+
+        [RequiresPreviewFeatures]
+        static byte IAdditionOperators<byte, byte, byte>.operator +(byte left, byte right)
+            => (byte)(left + right);
+
+        // [RequiresPreviewFeatures]
+        // static checked byte IAdditionOperators<byte, byte, byte>.operator +(byte left, byte right)
+        //     => checked((byte)(left + right));
```

Comment:
What are the functions commented out?  What's the plan for them?

Reply:
These are dependent on one or more of the proposed language features (such as `checked operators`) being approved and implemented. None of these were on the table for .NET 6.

Reply:
That's going to be a lot of additional commentation for likely little benefit.

For .NET 7, this will all be touched anyways to remove the `[RequiresPreviewFeatures]` annotations and making bits public, at which point these will either be uncommented (if they make it in) or removed (if LDM rejects the features).

Reply:
Fair enough. Thanks for considering it.

Reply:
My preference is to delete such code until such time that the language feature exists and we decide to and are able to add such features to the interfaces.  It's otherwise confusing to have such code in but commented out, especially without an explanation.